CandidateReranking.md

```diff
@@ -0,0 +1,368 @@
+| parsed_input | ParsedInput | **FALSE** | **ParsedInput** Object comprised of a Map<String,String> as `input_to_extracted_keywords` and a list of [ParsedEntities](#ParsedInput)
+| filters | [Filters[]](#Filter) | **FALSE** | Array of filters used in the original query to search.
+
+#### ParsedInput
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| input_to_extracted_keywords | Map<String, String> | **FALSE** | Words extracted from `search_keywords` as the **Key** and their relavtive keywords identified by SemanticSearchAPI as the **Value**.
+| parsed_entities | ParsedEntity | **FALSE** | Entities parsed from `search_keywords`.
+
+#### ParsedEntity
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| entity_name | String | **FALSE** | Name of the entity.
+| is_phrase | Boolean | **FALSE** | Flags the entity as a phrase or not.
+| is_selected | Boolean | **FALSE** | Flags the entity as sleceted or not.
+
+#### Keyword
+Entites returned from the SemanticSearchAPI.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| name | String | **FALSE** | Name of the keyword, e.g. "Java".
+| type | String | **FALSE** | Type of the keyword, e.g. "skill".
+| weight | Integer | **FALSE** | Weight of the keyword as an int. \[0-100\]
+| relationships | [Relationship](#relationship) | **FALSE** | Array of relationship objects which enrich the keyword with additional entities.
+
+#### Relationship
+A `relationship` is composed of a **\<semantic enrichment type\>** and the associated set of entities which can be classified under that **\<semantic enrichment type\>**. Accepted values for the **\<semantic enrichment type\>** are: `job_titles`, `occupations`, `skills`, `job_level`, `text_kernel_related_keywords`, `interesting_keywords`, `related_keywords`, `related_keywords_recruiter`, `related_keywords_jobseeker`, `raw_job_titles_jobseeker`, `raw_job_titles_recruiter`, `extracted_kewords`, `custom_keywords`, `related_search_terms`, and `job_titles_recruiter`.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| **\<semantic enrichment type\>** | [Entity[]](#entity) | **FALSE** | Relationship to the keyword `name` value.
+
+#### Entity
+An `entity` is composed of a `name`, `weight`, and `selected` boolean. An `entity` enriches the original keyword with terms or phrases which share a close association. For example, if you were to have a `keyword` java, an enrichment entity might be java or programming.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| name | String | **FALSE** | Name of enrichment `entity`. e.g., javascript, java, programming or software engineer.
+| weight | double | **FALSE** | Range: \[0.0 - 1.0\]. Weight of enriched term according to the Semantic Search API.
```

Comment:
Minor beef comment: could we put the range at the end to mirror how you specified it [here](https://github.com/careerbuilder/DataScienceAPIDocumentation/pull/136/files#diff-454d057d2679f3281a780face2d22ae4R78)?